tests/integration/pilot/istioctl_test.go

```diff
@@ -72,3 +117,60 @@ func TestVersion(t *testing.T) {
 			}
 		})
 }
+
+func TestDescribe(t *testing.T) {
+	framework.NewTest(t).
+		RunParallel(func(ctx framework.TestContext) {
+
+			ctx.NewSubTest("ISTIOCTL").
+				RequiresEnvironment(environment.Kube).
+				RunParallel(func(ctx framework.TestContext) {
+					ns := namespace.NewOrFail(ctx, ctx, namespace.Config{
+						Prefix: "istioctl-describe",
+						Inject: true,
+					})
+
+					deployment := file.AsStringOrFail(t, "../istioctl/testdata/a.yaml")
+					g.ApplyConfigOrFail(t, ns, deployment)
+
+					var a echo.Instance
+					echoboot.NewBuilderOrFail(ctx, ctx).
+						With(&a, echoConfig(ns, "a")).
+						BuildOrFail(ctx)
+
+					istioCtl := istioctl.NewOrFail(t, ctx, istioctl.Config{})
+
+					args := []string{fmt.Sprintf("--namespace=%s", ns.Name()),
+						"x", "describe", "svc", "a"}
+					output := istioCtl.InvokeOrFail(t, args)
+					g := gomega.NewGomegaWithT(t)
+					g.Expect(output).To(gomega.BeIdenticalTo(describeSvcAOutput))
+
+					podID, err := getPodID(a)
+					if err != nil {
+						t.Fatalf("Could not get Pod ID: %v", err)
```

Comment:
this needs to be the same thing that was passed to the func the tests runs in (`ctx` in this case)

Reply:
I don't understand the comment.  What is the problem?

Reply:
See the test failure: `testing.go:864: test executed panic(nil) or runtime.Goexit: subtest may have called FailNow on a parent test`

You can't use the parents `t`, its not a valid use of the go testing library